### DISCUSSION TITLE
Additional db options [Mar 05]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#5285] [Discord Threads]
> 🙋‍♂️ **replavar**  
## In the latest timely build you can no longer set the datatype for columns. Is that by design or an error? I hope it's an error:  
![image0](https://cdn.discordapp.com/attachments/661905455894888493/1070997962982953022/image.png)

**Comment by o1lab** [comment]
> 💁 **o1lab**  
#### if the table is on external db - its behind an easter egg   
#### double click on 'hide-all' - you should see it  
#  
> 🙋‍♂️ **replavar**  
#### Ah ok - I didn't think of that. Elegant way not to confuse some users but still give others the flexibility. Thanks!  
#  
> 💁 **o1lab**  
#### ya reason being, lot of non-dev tend to get confuse and mess it up a bit  
#  
> 💁 **o1lab**  
#### we will document that its behind easter egg  
#  
> 🙋‍♂️ **replavar**  
#### It makes sense to do it this way - good thinking!  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/1070997963184283668)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
